docs/ref/models/relations.txt

```diff
@@ -135,12 +135,31 @@ Related objects reference
         :class:`~django.db.models.ForeignKey`\s where ``null=True`` and it also
         accepts the ``bulk`` keyword argument.
 
+    .. method:: set(obj1, [obj2, ...])
```

Comment:
I wondered about it, we didn't do it for `bulk=True/False` so I didn't do it here. Should I add it?